gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -153,12 +124,105 @@ static GoogleCloudStorageGrpcReadChannel open(
     }
   }
 
+  private static GoogleCloudStorageGrpcReadChannel openChannel(StorageStubProvider stubProvider,
```

Comment:
nit: Would that be considered an improvement to have the input arguments one at a line?

Reply:
Agreed